new-pipeline/dist.html

```diff
@@ -55,8 +55,11 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                 </h2>
                 <hr></hr>
                 <figure class="col-md-12">
-                  <figcaption id="dist-caption"></figcaption>
-                    <div class="col-md-9" id="plots">
+                        <div class="caption-div">
+                            <figcaption id="dist-caption-text" class="overflow ellipsis"></figcaption>
+                            <figcaption id="dist-caption-link"></figcaption>
+                        </div>
+                        <div class="col-md-9" id="plots">
```

Comment:
This looks like the spaces are sorted now!
Now we just need to indent `dist-caption` and `col-md-9` one level less, as they're not on the same level as the following `<div class="row">`.